integration-tests/src/test/resources/integration-test/openmrs-distro.properties

```diff
@@ -13,3 +13,4 @@ spa.frontendModules.@openmrs/esm-login-app=3.3.1
 spa.frontendModules.@openmrs/esm-patient-chart-app=3.1.0
 spa.apiUrl=notopenmrs
 spa.configUrls=foo
+content.hiv=1.0.0
```

Comment:
@ibacher i added this so as to test whether [this test](https://github.com/openmrs/openmrs-sdk/blob/0285a8c64425246b647e65601d032b272c32e8c3/integration-tests/src/test/java/org/openmrs/maven/plugins/SetupIntegrationTest.java#L116) would pass and i ran it locally and it is passing. It is what i used to test the functionality on the `setup` command.